<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LTCToolWindow</class>
 <widget class="QMainWindow" name="LTCToolWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LTCToolWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="generateCheckBox">
      <property name="text">
       <string>Generate LTC</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="generatorGroupBox">
      <property name="title">
       <string>Generator:</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="generateInfoLabel">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>00:00:00:00 =&gt; 00:00:00:00</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="PhMediaPanel" name="widgetMaster" native="true"/>
       </item>
       <item>
        <widget class="QCheckBox" name="cBoxLoop">
         <property name="text">
          <string>Loop</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="readCheckBox">
      <property name="text">
       <string>Read LTC</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="readerGroupBox">
      <property name="title">
       <string>Reader:</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="readInfoLabel">
         <property name="text">
          <string>0x since 00:00:00:00</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblSlave">
         <property name="font">
          <font>
           <family>Arial Unicode MS</family>
           <pointsize>45</pointsize>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="text">
          <string notr="true">00:00:00:00</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="minMaxLevelLabel">
         <property name="text">
          <string>- / -</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>523</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuControls">
    <property name="title">
     <string>Controls</string>
    </property>
    <addaction name="actionSet_TC_In"/>
    <addaction name="actionSet_TC_Out"/>
    <addaction name="actionPreferences"/>
   </widget>
   <addaction name="menuControls"/>
  </widget>
  <action name="actionSet_TC_In">
   <property name="text">
    <string>Set TC In...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+I</string>
   </property>
  </action>
  <action name="actionSet_TC_Out">
   <property name="text">
    <string>Set TC Out...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PhMediaPanel</class>
   <extends>QWidget</extends>
   <header>PhCommonUI/PhMediaPanel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
